lib/std/build.zig

```diff
@@ -1816,7 +1816,7 @@ pub const LibExeObjStep = struct {
             },
             else => {},
         }
-        out.print("pub const {z} = {};\n", .{ name, value }) catch unreachable;
+        out.print("pub const {z}: {} = {};\n", .{ name, @typeName(T), value }) catch unreachable;
```

Comment:
This is fine but for future reference, I've discovered best practice to avoid bugs is:

```suggestion
        out.print("pub const {z}: {s} = {s};\n", .{ name, @typeName(T), value }) catch unreachable;
```